### ISSUE TITLE
Include thing/action object's vector position when ?meta=true

### ISSUE THREAD

**Issue by etiennedi** [semi-technologies/weaviate#1041]
## Proposal

When `?meta=true` is set on `GET /v1/things/<id>` we can include the raw vector position. This will help @fefi42 (and all users) debug classification performance.

## Todos
* [ ] include vector when `meta=true`

**Comment by etiennedi** [comment]
Any objections @bobvanluijt ?

**Comment by bobvanluijt** [comment]
Not only I don't object, but this is also great for demos ;-)

While you're on it, would it be possible to `/v1/c11y/concepts/{UUID}` to show relations etc?

**Comment by etiennedi** [comment]
> While you're on it, would it be possible to /v1/c11y/concepts/{UUID} to show relations etc?

Can you explain what that would do?

**Comment by laura-ham** [comment]
No objections, good idea!

**Comment by bobvanluijt** [comment]
> Can you explain what that would do? Also feel free to open a separate issue, I want to keep this one as small as possible, so I can unblock Felix asap.

Now, one gets a vector position for a concept by doing: `/v1/concepts/{word}` it would be awesome if this would also work for `/v1/concepts/{UUID}`.

@laura-ham I think this might also be great for the docs, this might allow people to build ML models outside Weaviate.

**Comment by bobvanluijt** [comment]
Addition: can we also show the nearest neighbors like the c11y/concepts endpoint?

**Comment by etiennedi** [comment]
Sure, something like 
`GET /v1/things/{uuid}/neighbors` sounds like the perfect place to put them.

What would the neighbors contain? Only neighboring objects? Or neighboring words? Or both?

If we inlcude objects, it's essentially the same (which isn't negative, just want to highlight it) as #1085

**Comment by bobvanluijt** [comment]
Actually, the exact same way as described in [this answer](https://github.com/semi-technologies/weaviate/issues/1073#issuecomment-586745639).

**Comment by etiennedi** [comment]
Scoped the "neighbors" feature out to #1098, so we can close this issue as soon as we have vector position in the object's meta.
